src/Readers/GithubReader.js

```diff
@@ -131,17 +132,22 @@ class GithubReader extends AbstractReader {
    * @param {string} source
    */
   static fetch(source, username, password) {
-
+    var agent = null;
+    if (process.env.HTTPS_PROXY) {
+      agent = HttpsProxyAgent(process.env.HTTPS_PROXY);
+    } else if (process.env.https_proxy) {
+      agent = HttpsProxyAgent(process.env.https_proxy);
+    }
     const github = new GitHubApi({
       version: '3.0.0',
```

Comment:
Should the version be updated?

Reply:
Current version of GitHub API lib doesn't require this parameter, so I removed it.